2018/index.dd

```diff
@@ -42,6 +42,12 @@ $(P
 Building on the success of the past year's hackathon, we are happy to announce a day-long Hackathon on Saturday, May 5th. The day is largely unstructured, but we expect birds-of-a-feather groups to materialize naturally. Bring your laptop and your enthusiasm!
 )
 
+$(T h2,, Sponsorship)
+
+$(P
+Sponsorship packages are available. For details, please direct queries to Franziska Mai ($(LINK2 mailto:f.mai@hlmc.de, f.mai@hlmc.de)) or Gerhard Versteegen ($(LINK2 mailto:g.versteegen@hlmc.de, g.versteegen@hlmc.de)).
```

Comment:
Should we escape the mail addresses to prevent them from being easily found by spammers?